go.mod

```diff
@@ -1,23 +1,23 @@
 module github.com/bufbuild/knit-demo
 
-go 1.19
+go 1.22.0
```

Comment:
This seems awfully high. Why jump all the way to the latest? In all of our other Go repos, we recently updated to a minimum of 1.20.

The CI workflow in here is using Go 1.21. Maybe update this to that version, 1.21, and hold off on going to 1.22 until we actually need to?